.github/workflows/rust.yml

```diff
@@ -122,6 +122,8 @@ jobs:
       - uses: Swatinem/rust-cache@v2
       - run: cargo build --locked --all-targets
       - run: cargo test --locked
+      - run: cargo test --locked -p quinn-udp --features fast-apple-datapath
+        if: ${{ runner.os }} == "macOS"
```

Comment:
Further below we use `${{ matrix.os }} == "macos-latest"`. Should we be consistent, either changing here or below?

Reply:
I suggest changing the entry below, i.e. running the benchmark across all MacOS versions. Happy to do a follow-up pull request in case you prefer not doing it here.

Reply:
Updated!